portfolio/src/main/webapp/script.js

```diff
@@ -122,23 +127,21 @@ function deleteTextFromDom(text) {
 
 function setCurrentPage() {
     const current = document.getElementById("current-page");
-    if (sessionStorage["current-page"]) {
-        current.innerText = sessionStorage["current-page"];
-    } else {
-        sessionStorage["current-page"] = 1;
-    }
+    const defaultPage = 1;
+    sessionStorage["current-page"] = sessionStorage["current-page"] || defaultPage;
+    current.innerText = sessionStorage["current-page"];
```

Comment:
Shouldn't this call getMessages or something to update the comments visible?

Reply:
This is called in changeButtonValUp and changeButtonValDown which submit a form, so the page refreshes